index.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Weather Dashboard Project</title>
```

Comment:
@dosanjhkiran Can you think of a new title?

Reply:
We can add the title later when it's finalised. :)